Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.h

```diff
@@ -276,6 +276,9 @@ public slots:
   /// Enable/disable masking using master volume intensity
   void toggleMasterVolumeIntensityMaskEnabled();
 
+  /// Enable/disable masking using master volume intensity
```

Comment:
Wrong doc text

Reply:
I would probably just use it for some seconds to see the full range of effects, then turn it off again to save space.